libs/index/odc/index/stac.py

```diff
@@ -9,16 +11,35 @@
     'sentinel-2'
 ]
 
+LANDSAT_PLATFORMS = [
+    'landsat-5', 'landsat-7', 'landsat-8'
+]
 
-def _stac_product_lookup(item):
+# Mapping between EO3 field names and STAC properties object field names
+MAPPING_STAC_TO_EO3 = {
```

Comment:
@alexgleith So what's up with these mappings. Have there been update to STAC and those fields are now standard? Cause then we should probably update eo3 instead. This will be fine for now, but not in a long term.